Documentation/gettingstarted/k8s-install-azure.rst

```diff
@@ -102,3 +104,14 @@ Limitations
 
 * All VMs and VM scale sets used in a cluster must belong to the same resource
   group.
+
+.. _azure_troubleshooting:
+
+Troubleshooting
+===============
+* If ``kubectl exec`` to a pod fails to connect, restarting ``tunnelfront`` POD may help.
```

Comment:
Nit:
```suggestion
* If ``kubectl exec`` to a pod fails to connect, restarting the ``tunnelfront`` POD may help.
```

I also think that we don't use the uppercase POD spelling anywhere else. Ususally we seem to use either "pod" or "Pod".

Reply:
Applied, thanks!